ampl-converter/src/test/resources/outputs/svc.txt

```diff
@@ -1,3 +1,3 @@
 #Shunts
 #"num" "bus" "v regul" "targetV (pu)" "Q (MVar)"
-1 1 false 400.000 30.0000
+1 1 false 40.0000 30.0000
```

Comment:
Just a small remark: realistic values for the "target V" would be around 1 (0.9 to 1.1 typically).
Not blocking at all for this unit test.

Reply:
Agree: Could you please change the value to something closer to a real value: the unit test will be more understandable with values close to reality.